### PR TITLE
Fixes #21866 {resourcename.isnot.unique}  message key is shown when create-connect…

### LINKED ISSUE #21866: {resourcename.isnot.unique} message key is shown when create-connector-resource with same resource name as other pool name.
{resourcename.isnot.unique} message key is shown when create-connector-resource with same resource name as other pool name.

## Environment Details
* GlassFish Version: 5.0-b08
* JDK version: 1.8.0_65
* OS: Windows 7

----------

## Problem Description
When I execute asadmin create-connector-resource subcommand with same resource name as other jdbc-connection-pool name or connector-connection-pool name, error message shows violation reason is [ {resourcename.isnot.unique} ].

```
> asadmin create-connector-resource --poolname jms/__defaultConnectionFactory-Connection-Pool DerbyPool
remote failure: Unable to create connector resource DerbyPool. Constraints for this ConnectorResource configuration have been violated: on property [  ] violation reason [ {resourcename.isnot.unique} ]
Command create-connector-resource failed.
```

{resourcename.isnot.unique} is a message key in LocalStrings.properties file, so it should be translated into "Name is already used by another resource." message. But the message key isn't translated.

In appserver/connectors/connectors-internal-api project, there are two LocalStrings.properties in java and resources directory.
1. src/main/java/org/glassfish/connectors/config/LocalStrings.properties
2. src/main/resources/org/glassfish/connectors/config/LocalStrings.properties

Built module "connectors-internal-api.jar" contains only 1. LocalStrings.properties.
But {resourcename.isnot.unique} message key is defined only 2. LocalStrings.properties.
So the message key isn't translated.

## Steps to reproduce
1. asadmin start-domain
1. asadmin create-connector-resource --poolname jms/__defaultConnectionFactory-Connection-Pool DerbyPool

## Fix
Merging two LocalStrings.properties would fix this issue.
LocalStrings.properties is a resource file, so it should be in resources directory (delete 1. and merge into 2.).

### PR DESCRIPTION
**Pull request by mukumawa**
…or-resource with same resource name as other pool name.

Fixes #21866

### CODE REVIEW

File: appserver/connectors/connectors-internal-api/src/main/resources/org/glassfish/connectors/config/LocalStrings.properties [hunk 1]
```diff
@@ -39,3 +39,7 @@
 #
```
Comment (arindam-bandyopadhyay) [inline appserver/connectors/connectors-internal-api/src/main/resources/org/glassfish/connectors/config/LocalStrings.properties]: Why copyright was removed?

Comment (mukumawa) [inline appserver/connectors/connectors-internal-api/src/main/resources/org/glassfish/connectors/config/LocalStrings.properties]: I have removed the file.

Comment (arindam-bandyopadhyay) [inline appserver/connectors/connectors-internal-api/src/main/resources/org/glassfish/connectors/config/LocalStrings.properties]: sorry .. overlooked

**Comment by mukumawa** [comment]
@glassfishrobot Run CI tests please

**Review by yaminikb — APPROVED** [review]

**Pull request merged**